tests/Utils/CoerceValueTest.php

```diff
@@ -375,11 +392,11 @@ public function testReturnsErrorForIncorrectValueType(): void
     }
 
     /**
-     * @see it('returns no error for a valid input')
```

Comment:
please self-review and revert changes like this

Reply:
I can `vendor/bin/php-cs-fixer fix` before committing... seems its not working :shrug: